src/traces/parcoords/attributes.js

```diff
@@ -47,6 +49,10 @@ module.exports = {
         }
     },
 
+    labelfont: extendFlat({}, fontAttrs, {dflt: {size: 10}, description: 'Sets the font for the `dimension` labels.'}),
```

Comment:
The purpose was to have _smaller_ fonts than what would come from the `layout` defaults, which would result in overly big numbers, looked really bad. The `dimension` labels and tick numbers need to be quite small by default, e.g. around 10px. It's achieved by `parcoords/attributes` specifying these sizes. Is there another way of saying, 'take layout font values but don't adopt large fonts unless the user wants it, in which case they specify it on eg. `labelfont`?